eventfiles/models.py

```diff
@@ -8,17 +8,32 @@ def upload_to_function(instance, filename):
     return path.join('files', str(instance.participant_id.event_id.id), path.basename(filename))
 
 
+class EventFileQuerySet(models.QuerySet):
+    def get_files_by_participant(self, participant):
+        return self.filter(participant_id=participant)
+
+    def get_files_by_event(self, event_id):
+        participants = EventParticipant.objects.get_an_event_participants(event_id)
+        files = []
+        for participant in participants:
+            files.extend(self.get_files_by_participant(participant))
+
+        return files
```

Comment:
```suggestion
    def get_files_by_event(self, event_id):
        return self.filter(participant_id__event_id=event_id)
```

In my opinion, this is a more readable and 'elegant'  way to perform this task

Reply:
Done